### DISCUSSION TITLE
Creating an "Add inbox rule" from "Reply to" field

### DISCUSSION THREAD

**Discussion by serrq** [tutao/tutanota#6439] [Ideas]
I am in a very special case: my sender is hidden behind a "service", so its "From" field is the same for thousands of other news sources, and its (technical) "Sender" field is new in each email I receive from it.

Solution: attacking the "Reply to" field creating a rule beginning from that field.

**Comment by charlag** [comment]
You can create an inbox rule based on a header, does this help? it's a bit of a workaround though

**Reply by serrq** [reply]
How?
